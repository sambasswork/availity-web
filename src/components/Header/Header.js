import React from 'react';
import { Navbar } from 'react-bootstrap';
import logo from './AvailityWhiteLogo.png';
import styled from 'styled-components';

const HeaderStyle = styled.div`
    background: linear-gradient(90deg,#ff8d00 24%,#fce404 100%);
`;

export function Header() {
    return(
        <HeaderStyle>
        <Navbar>
            <Navbar.Brand href="#home">
                <img
                    alt="logo"
                    src={logo}
                    width="140"
                    className="d-inline-block align-top"
                />
            </Navbar.Brand>
        </Navbar>
    </HeaderStyle>)
}