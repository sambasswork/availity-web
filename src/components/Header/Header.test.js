import React from 'react';
import renderer from 'react-test-renderer';
import { Header } from './Header';

describe('testing Header component', () => {
    it('if Header component renders correctly', () => {
        const tree = renderer
            .create(<Header />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
})