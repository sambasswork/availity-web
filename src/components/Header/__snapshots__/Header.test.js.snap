// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Header component if Header component renders correctly 1`] = `
<div
  className="sc-AxjAm bsDrnh"
>
  <nav
    className="navbar navbar-expand navbar-light"
  >
    <a
      className="navbar-brand"
      href="#home"
    >
      <img
        alt="logo"
        className="d-inline-block align-top"
        src="AvailityWhiteLogo.png"
        width="140"
      />
    </a>
  </nav>
</div>
`;
